inc/saiswitch.h

```diff
@@ -472,6 +472,17 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_DEFAULT_VIRTUAL_ROUTER_ID,
 
     /**
+     * @brief Default .1Q Bridge ID
```

Comment:
no. we still have the default vlan 1, and all ports joined the vlan 1 by default. The behavior is the same as before.